src/odemis/driver/hamamatsurx.py

```diff
@@ -1120,15 +1118,18 @@ def readCommandResponse(self):
                         # Use nasty trick to work around for this command.
                         # This command first receives the error_code and then additional information.
                         # The total information has a fixed length of 5 lines:
-                        # raw output: TODO check on HW! check how lines are separated
+                        # rargs: 'OrcaFlash 4.0 V3'
+                        # additional_info:
+                        # '\nProduct number: C13440-20C\r\nSerial number: 301730\r\nFirmware: 4.20.B\r\nVersion: 4.20.B03-A19-B02-4.02\r'
```

Comment:
If indeed, the end of the lines finishes with `\r` only, while the in-between lines finish with `\r\n` it should be at least possible to automatically detect the end of the transmission (instead of always waiting for 1sec).
Probably, it should also be possible to avoid all the special `self._getReadoutCamInfo` by detecting that the first `\r` is not the end, because it's immediately followed by `\n`. Just need to be a little careful in case the last character received is `\r` only because the receive buffer was full, and actually reading further would bring `\n` (that's a very unlikely case, as typically the answers are much shorter than the buffer).

Reply:
We leave it for now as responses is usually empty and then the rest of the info is sent. This function is anyways only called at initialization, so we don't care for now if we have to wait for another second.